src/Microsoft.AspNetCore.Mvc.Core/ApplicationModels/ApiBehaviorApplicationModelProvider.cs

```diff
@@ -66,8 +67,6 @@ internal class ApiBehaviorApplicationModelProvider : IApplicationModelProvider
 
         public List<IActionModelConvention> ActionModelConventions { get; }
 
-        public List<IControllerModelConvention> ControllerModelConventions { get; }
```

Comment:
I can see we don't need `IControllerModelConvention`s anymore. But, what about our customers? Suggest leaving the property and the application of any items in the list. That is, limit change to initializing it as empty.

If this is a removal we've already decided on, remove the `IControllerModelConvention` interface as well.

Reply:
This type is internal, so this does not affect users.